modules/caddyhttp/logging.go

```diff
@@ -80,10 +80,11 @@ func (slc ServerLogConfig) wrapLogger(logger *zap.Logger, req *http.Request) []*
 	}
 	hosts := slc.getLoggerHosts(hostWithoutPort)
 	loggers := make([]*zap.Logger, 0, len(hosts))
-	if access_logger_names := GetVar(req.Context(), AccessLoggerNameVarKey); access_logger_names != nil {
-		accessLoggerNamesSlice := strings.Split(access_logger_names.(string), ",")
-		for _, loggerName := range accessLoggerNamesSlice {
-			loggers = append(loggers, logger.Named(loggerName))
+	if names := GetVar(req.Context(), AccessLoggerNameVarKey); names != nil {
+		if namesSlice, ok := names.([]any); ok {
```

Comment:
Should this type assert as `[]string` instead? Then you don't need to do `.(string)` later, I think.

Reply:
I'm wrong, that doesn't work. Go is silly, can't cast directly to a slice type.